pyeudiw/tests/oauth2/test_dpop.py

```diff
@@ -65,18 +65,40 @@ def wia_jws(jwshelper):
 
 def test_create_validate_dpop_http_headers(wia_jws, private_jwk=PRIVATE_JWK):
     # create
-    unpad_jwt_header(wia_jws)
+    header = unpad_jwt_header(wia_jws)
+    assert header
+    assert header["trust_chain"] == []
+    assert header["x5c"] == []
+    assert header["alg"]
+    assert header["kid"]
+    
     payload = unpad_jwt_payload(wia_jws)
-    # TODO assertions for upadded headers and payload
-
+    assert payload
+    assert payload["iss"] == WALLET_INSTANCE_ATTESTATION["iss"]
+    assert payload["sub"] == WALLET_INSTANCE_ATTESTATION["sub"]
+    assert payload["type"] == WALLET_INSTANCE_ATTESTATION["type"]
+    assert payload["policy_uri"] == WALLET_INSTANCE_ATTESTATION["policy_uri"]
+    assert payload["tos_uri"] == WALLET_INSTANCE_ATTESTATION["tos_uri"]
+    assert payload["logo_uri"] == WALLET_INSTANCE_ATTESTATION["logo_uri"]
+    assert payload["asc"] == WALLET_INSTANCE_ATTESTATION["asc"]
+    assert payload["cnf"]
+    assert payload["cnf"]["jwk"]
+    assert payload["cnf"]["jwk"]["kty"]
+    assert payload["cnf"]["jwk"]["crv"]
+    assert payload["cnf"]["jwk"]["kid"]
+    assert payload["cnf"]["jwk"]["x"]
+    assert payload["cnf"]["jwk"]["y"]
+    assert payload["authorization_endpoint"] == WALLET_INSTANCE_ATTESTATION["authorization_endpoint"]
+    assert payload["response_types_supported"] == WALLET_INSTANCE_ATTESTATION["response_types_supported"]
+    assert payload["vp_formats_supported"] == WALLET_INSTANCE_ATTESTATION["vp_formats_supported"]
+   
     new_dpop = DPoPIssuer(
         htu='https://example.org/redirect',
         token=wia_jws,
         private_jwk=private_jwk
     )
     proof = new_dpop.proof
-
-    # TODO assertions
+    assert proof
```

Comment:
assert proof alone is not enough

please do tests of the ath hash over the WIA